design-proposals/co-cli.md

```diff
@@ -0,0 +1,101 @@
+# Design Proposal: Command Line Interface for Cluster Orchestration components
+
+Author(s): Julia Okuniewska
+
+Last updated: 2025-04-30
+
+## Abstract
+
+This ADR describes the design of nouns supported by cluster-orchestration
+in command line interface (CLI).
+
+This is a work-in-progress document that is complementary to general CLI design:
+https://github.com/open-edge-platform/edge-manageability-framework/pull/246.
+
+## Proposal
+
+The CLI follows a `verb - noun - subject` pattern.
+
+The target for cluster-orchestration part in CLI is to be able to:
+- create/list/delete clustertemplates
+- mark clustertemplate as default one
+- create cluster with given hosts and given template (or pick default template if not specified)
```

Comment:
Although not specifically mentioned here, I believe delete cluster should also be supported

Reply:
Yes, info about get/delete added here 
https://github.com/open-edge-platform/edge-manageability-framework/pull/251/commits/006be86051c530e04db93ed5745c2fdfaea778d4